src/app/public/modules/radio/radio.component.ts

```diff
@@ -46,6 +46,11 @@ const SKY_RADIO_CONTROL_VALUE_ACCESSOR: Provider = {
 };
 // tslint:enable
 
+/**
+ * The radio button component renders a SKY UX-themed replacement for an HTML `input` element
```

Comment:
Looks like this wasn't addressed. Not a huge deal, but just pointing it out so that we can either address it or resolve this comment thread as "will not address."

Reply:
Done